Source/Initialization/PlasmaInjector.H

```diff
@@ -39,6 +39,8 @@ public:
 
     bool insideBounds (amrex::Real x, amrex::Real y, amrex::Real z) const noexcept;
 
+    bool overlapsWith (const amrex::XDim3& lo, const amrex::XDim3& hi) const noexcept;
```

Comment:
```suggestion
    // bool: whether the region defined by lo and hi overaps with the plasma region
    bool overlapsWith (const amrex::XDim3& lo, const amrex::XDim3& hi) const noexcept;
```

Reply:
A small typo in the comment : overaps instead of overlaps